features/step_definitions/my_setps.rb

```diff
@@ -0,0 +1,32 @@
+Given("the following user exists:") do |table|
+  table.hashes.each do |table|
+    User.create(table)
+  end
+end
+
+Given("I am on the landing page") do
+  visit '/'
+end
+
+When("I click on the link {string}") do |string|
+  click_on(string)
+end
+
+When("I fill in {string} with {string}") do |a, b|
+  fill_in(a, :with => b)
```

Comment:
never use single letter variable names this is not mathematics. In order to make your code more readable use proper naming.